### PR TITLE
Fixed errors in client

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: client/src/app/core/repositories/users/user-repository.service.ts [hunk 1]
```diff
@@ -500,6 +500,9 @@ export class UserRepositoryService extends BaseRepository<ViewUser, User, UserTi
     }
 
     private preventAlterationOnDemoUsers(users: ViewUser | ViewUser[]): void {
+        if (!this.demoModeUserIds) {
+            return;
+        }
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/users/user-repository.service.ts:505]: That's dirty. Rather have:
```
if (this.demoModeUserIds) {
    ...
}
```
and do not return anything in void functions

File: client/src/app/core/ui-services/load-font.service.ts [hunk 2]
```diff
@@ -68,6 +68,9 @@ export class LoadFontService {
      */
     private setCustomProjectorFont(font: any, weight: number): void {
         const path = font.path ? font.path : font.default;
+        if (!path) {
+            return;
+        }
```
Comment (tsiegleauq) [inline client/src/app/core/ui-services/load-font.service.ts:73]: Same as above:
`if (path) { ... }`

**Review by tsiegleauq — CHANGES_REQUESTED** [review]

File: client/src/app/core/repositories/users/user-repository.service.ts (hunk 1 above)
Comment (FinnStutzenstein) [reply client/src/app/core/repositories/users/user-repository.service.ts:505]: -> Early return pattern?

> and do not return anything in void functions

I do not return anything, right?

Comment (tsiegleauq) [reply client/src/app/core/repositories/users/user-repository.service.ts:505]: >  Early return pattern?

It's really ugly. You may validate that data is correct, not wrong. I'd prefer the optional chaining method as in the other pr.

> I do not return anything, right?

Do not use the `return` statement in void function

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**